eg-api.php

```diff
@@ -181,29 +183,30 @@
 		//error_log("cleaning up files");
 		cleanupFiles($files);
 		//error_log("done writing to db");
-	}// end of processing req from a valid user
 
 
-	error_log("checking whether to email petitions.");
+		error_log("checking whether to email petitions.");
+		
+		if (isset($request['emailPetitions']) && preg_match('/^(t|true|1)$/i', $request['emailPetitions'])===1){
+			$log_trail .= ",emailing results";
+			if (!(isset($request['createPetitions']) && preg_match('/^(t|true|1)$/i', $request['createPetitions'])===1)) {
+				$file_path = NULL;
+				if (isset($response['results']['expungeZip'])) {
+					unset($response['results']['expungeZip']);
+				}
+			} else { 
+				$file_path = $response['results']['expungeZip'];
+				$path_parts = pathinfo($response['results']['expungeZip']);
+				$response['results']['expungeZip'] = $baseURL . "secureServe.php?serveFile=" . $path_parts['filename']; 
+			} 
+		    //mailPetition($_REQUEST['current_user'], $_REQUEST['current_user'], $response, $file_path);
+		    error_log("Mailing to " . mailDestination($request));
+		    mailPetition(mailDestination($request), mailDestination($request), $response, $file_path);
+		} else {
+			error_log("emailPetitions was not set");
+		}
 	
-	if (isset($_REQUEST['emailPetitions']) && preg_match('/^(t|true|1)$/i', $_REQUEST['emailPetitions'])===1){
```

Comment:
I changed this stuff because I think the indentation was wrong here. This stuff should only be reached in a valid request, but because of how it was indented and the location of the close bracket on line 209, invalid requests were getting checked for emailing. I think. Kind of a macro project for *someday* is break this up into smaller pieces so that its less of a complicated nest of brackets.